packages/core/src/routes/applications/application.ts

```diff
@@ -81,7 +78,7 @@ export default function applicationRoutes<T extends AuthedRouter>(
         excludeRoleId: string().optional(),
         isThirdParty: z.union([z.literal('true'), z.literal('false')]).optional(),
       }),
-      response: z.array(applicationResponseGuard),
+      response: z.array(Applications.guard),
```

Comment:
Do we need to filter protected apps for OSS?

Reply:
I don't think so, the API docs says this is not for open source version. If we want to make different types for different environment, it would be a big project.